src/GurobiSolverInterface.jl

```diff
@@ -127,7 +127,7 @@ function loadproblem!(m::GurobiMathProgModel, A, collb, colub, obj, rowlb, rowub
   # check if we have any range constraints
   # to properly support these, we will need to keep track of the
   # slack variables automatically added by gurobi.
-  rangeconstrs = sum((rowlb .!= rowub) & (rowlb .> neginf) & (rowub .< posinf))
+  rangeconstrs = sum((rowlb .!= rowub) .& (rowlb .> neginf) .& (rowub .< posinf))
```

Comment:
Yes. It is supported by Compat v0.13: https://github.com/JuliaLang/Compat.jl/commit/4e49ddbce1e4b575deb5c977c301bdbd3315d8be